axum-extra/src/extract/json_deserializer.rs

```diff
@@ -10,7 +10,7 @@ use std::marker::PhantomData;
 
 /// JSON Extractor for zero-copy deserialization.
 ///
-/// Deserialize request bodies into some type that implements [`serde::Deserialize<'de>`].
+/// Deserialize request bodies into some type that implements [`serde::Deserialize<'de>`](`serde::Deserialize`).
```

Comment:
This is not what I suggested, can you confirm that what I wrote also works?

```suggestion
/// Deserialize request bodies into some type that implements [`serde::Deserialize<'de>`][serde::Deserialize].
```

I would prefer it because round-bracket links are more commonly used for regular HTTP links (not touched by rustdoc), and the extra backticks seem pointless.

Reply:
yes this is also working